### PR TITLE
fixes a few minor typos and wording

### PR DESCRIPTION
**Pull request by shelane**
I noticed you made some changes, so I reread this page and found a couple of minor changes needed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Docker powered web development environments for macOS, Windows, and Linux. Unifi
 - Full power of Docker Compose to suit any project without artificial syntax limitations
 - Unprecedented ability to automate routine tasks, slashing the time to on-board new team members.
 
-    Create your [custom automation](https://docs.docksal.io/fin/custom-commands/) in Bash, PHP or Node
```
Comment (achekulaev) [inline README.md:14]: nope

File: README.md [hunk 2]
```diff
@@ -23,19 +23,19 @@ and others is one step away with ready-to-use boilerplates.
...
+[Install Docksal](https://docs.docksal.io/getting-started/setup/), cd to the desired directory, and run:
 
 ```bash
 fin project create
 ```
 
-The wizard will guide you throught creation of new project using any of Instant Boilerplate Projects below. 
-The wizard will create the proper Docker containers stack depending on the project choice, and install 
-a working empty project preconfigured for running in Docksal, that you can build upon.
+The wizard will guide you throughout the creation of a new project using any of the boilerplate projects listed below.
```
Comment (achekulaev) [inline README.md:32]: Keep "Instant Boilerplate Projects" phrase please

File: README.md [hunk 3]
```diff
@@ -23,19 +23,19 @@ and others is one step away with ready-to-use boilerplates.
...
+The wizard will create the proper Docker containers stack, depending on the project choice, and install a working empty project
```
Comment (achekulaev) [inline README.md:33]: Don't see a reason for change in line break, it's minor can be kept as is

File: README.md [hunk 4]
```diff
@@ -23,19 +23,19 @@ and others is one step away with ready-to-use boilerplates.
...
+preconfigured for running in Docksal that you can build upon.
 
 ## Instant Boilerplate Projects
 
-In case you want to be have full control, you can clone one the boilerplate projects below and run `fin init`.
+In case you want to have full control, you can clone one the boilerplate projects below and run `fin init`.
```
Comment (achekulaev) [inline README.md:38]: I meant to write "to be in full control". Have should be deleted

File: README.md (hunk 1 above)
Comment (shelane) [reply README.md:14]: Ah, I see.

**Comment by shelane** [comment]
Changes made.

File: README.md (hunk 1 above)
Comment (achekulaev) [reply README.md:14]: Could you roll in back?

**Comment by achekulaev** [comment]
@shelane rollback the change in the first comment please

File: README.md (hunk 1 above)
Comment (shelane) [reply README.md:14]: I don't understand what you're asking me to do. I added the spaces back and the only difference in this line now is the extra comma that I added.

Comment (achekulaev) [reply README.md:14]: Oh I see, thanks

**Pull request merged**